src/Assignment-2/assignment2.html

```diff
@@ -14,6 +14,139 @@
 	<!-- Start your code from here -->
 	
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+	<div class="main-body">
+        
+        <!-- START Header -->
+        <div>
+            <header>
+                <div class="first-part">
+                    <div class="item">
+                        <img src="https://www.the10xacademy.com/assets/images/logo-color.png" alt="">
+                    </div>
+                    <div class="item">  
+                        <form action="">
+                            <input type="text" placeholder="Search" name="Search">
+                        </form>
+                    </div>
+                </div>
+                <div class="second-part">
+                    <nav class="nav">
+                        <div class="nav-link"><a href="https://github.com/mrunabhavsar28">My Github Link</a></div>
+                        <div class="nav-link"><a href="https://www.the10xacademy.com/">The 10xacademy</a></div>
+                        <div class="nav-link"><a href="https://www.hackthebox.com/">Hackthebox</a></div>
+                        <div class="nav-link"><a href="https://www.pexels.com/">Pexels</a></div>
+​
+                    </nav>
+                </div>
+            </header>
+        </div>
+        <!-- END Header -->
+    <hr>
+​
+        <!-- START BODY -->
+        <section class="main-section">
+            <div class="left-section">
+                    <aside>
+                        <ul>Section 1</ul>
+                        <ul>Section 2</ul>
+                        <ul>Section 3</ul>
+                        <ul>Section 4</ul>
+                        <ul>Section 5</ul>
+                        <ul>Section 6</ul>
+                        <ul>Section 7</ul>
+                        <ul>Section 8</ul>
+                        <ul>Section 9</ul>
+                        <ul>Section 10</ul>
+                    </aside>
+            </div>
+​
+            <!-- START MID -->
+            <section class="left-section">
+                    <section class="post-img">
+                        <img src="https://images.pexels.com/photos/5389628/pexels-photo-5389628.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" alt="">
+                        </section>        
+                        
+                        <section class="post-img">
```

Comment:
its not good practice to use many section tags. keep all the image tags in one section tag